EIPS/eip-7400.md

```diff
@@ -0,0 +1,211 @@
+---
+eip: 7400
+title: Flash Loans
+author: Alberto Cuesta Cañada (@alcueca), (@ultrasecreth)
```

Comment:
```suggestion
author: Alberto Cuesta Cañada (@alcueca)
```
Please only use a valid GitHub username.  You can list an author by name without contact information.

Reply:
That's a valid github username, do you mean I should do it like this?

`author: Alberto Cuesta Cañada (@alcueca), ultrasecreth (@ultrasecreth)`